cigaradvisor/styles/styles.css

```diff
@@ -142,6 +142,7 @@
   --light-grey: #d3d3d3;
   --dark-grey: #3c3a3b;
   --off-white: #f7f7f7;
+  --section-background: #eaeaea;
```

Comment:
What color is this? `section-background` isn't clear - in the context of the use (below) it's for a section with a grey background, let's make that the variable name.